site/content/en/docs/Advanced/high-availability-agones.md

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
Also, our site docs are continuously pushed. Since we're doc'ing a feature for a future release (it's not in Alpha in 1.29), you would normally use `publishVersion` (see https://agones.dev/site/docs/contribute/#within-a-page). It looks like at the page level that's not available, so you can use: https://agones.dev/site/docs/contribute/#at-the-page-level

Reply:
Added the `publishDate` for Feb 28.